_data/simple-icons.json

```diff
@@ -7996,6 +7996,11 @@
             "hex": "3A98D9",
             "source": "https://storify.com"
         },
+        {
+            "title": "Storyblok",
+            "hex": "0AB3AF",
+            "source": "https://www.storyblok.com/"
```

Comment:
The source should probably be https://www.storyblok.com/press. Within that ZIP I used `colored-standalone.svg` for comparison. I'd also suggest we add the link to the 'guidelines' entry as there's the line about requesting permission to use it.

Reply:
Updated. Not sure about what you are exactly asking with the link to the guidelines entry. What link should I use as a source?

Reply:
So you can add the press page as both the 'source' and 'guidelines' entry in the JSON, as below:

```
{
	"title": "Storyblok",
	"hex": "0AB3AF",
	"source": "https://www.storyblok.com/press",
	"guidelines": "https://www.storyblok.com/press"
}
```

It doesn't matter about the duplication - so don't worry about that. Just as I mentioned above, with them putting the line about requesting permission to use the icon - I think we're safest to include it.

Reply:
Ok, I understand. Fixed, BTW.